click_command_tree.py

```diff
@@ -30,7 +30,8 @@ def _build_command_tree(click_command):
 
     if isinstance(click_command, click.core.Group):
         for _, cmd in click_command.commands.items():
-            wrapper.children.append(_build_command_tree(cmd))
+            if not cmd.hidden: 
```

Comment:
It looks like the `hidden` attribute was added in v7, and I would like to keep this library as backwards compatible as possible. Could this also handle the case where this attribute is not there?

```suggestion
            if not getattr(cmd, "hidden", False): 
```

Reply:
Sure,
 the offered solution was just proof of concept. Thank you for reply